docs/reference/objects/product/variant/index.md

```diff
@@ -225,7 +239,9 @@ The variant's current active [promotion]({% link docs/reference/objects/product/
 number
 {: .label .fs-1 }
 
-The remaining stock for the variant, if the variant has infinite inventory this will return `nil`.
+The remaining stock for the variant, if the variant has infinite stock this will return `nil`.
+- When accessed through an [experience date]({% link docs/reference/objects/product/experience_date.md %}), it will return the remaining stock for the variant on the given date.
+- When accessed independently or through a product, it will return the remaining stock for the variant on the next upcoming date (if there is one) or the most recent date.
```

Comment:
Should this now be updated based on this thread? https://easolhq.slack.com/archives/C03AJ5F6L1L/p1689348944793599